sbtc/src/error.rs

```diff
@@ -70,12 +70,16 @@ pub enum Error {
     },
 
     /// This is thrown when failing to parse a hex string into bytes.
-    #[cfg(feature = "webhooks")]
+    #[cfg(any(test, feature = "webhooks"))]
     #[error("could not decode the hex string into bytes: {0}")]
     DecodeHexBytes(#[source] hex::FromHexError),
+    /// This is thrown when failing to parse a hex string into bytes.
```

Comment:
Nit: I think it is not nice to have two identical comments for two different error variants